lib/app-layout/index.tsx

```diff
@@ -141,15 +135,10 @@ export class AppLayout extends Component<Props> {
             >
               <NoteToolbar aria-hidden={hiddenByRevisions} />
               {showRevisions ? (
-                <NoteRevisions
-                  aria-hidden={hiddenByRevisions}
-                  noteId={openedNote}
-                  note={openedRevision}
```

Comment:
I saw that these values are only used for this component so since the can be fetched from the state in the `connect` of `NoteRevisions` component, I decided to remove them to keep this component cleaner.